addons/base_install_request/wizard/base_module_install_request_views.xml

```diff
@@ -13,9 +13,11 @@
             <form>
                 <field name="module_id" invisible="1"/>
                 <group string="Send to:">
+                    <label for="user_ids" string=""/>
                     <field name="user_ids" widget="many2many_tags" nolabel="1"/>
```

Comment:
Have you tried a `colspan="2"` on the `<field>`...

Reply:
I haven't. I'll try that.

Reply:
Works like charm. Thanks. :)